test/land/create_bkg_ens.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
+################################################
+# 4. CREATE BACKGROUND ENSEMBLE (LETKFOI)
+################################################
+YYYY=2021
+MM=03
+DD=23
+HH=18
+FILEDATE=$YYYY$MM$DD.${HH}0000
+
+project_binary_dir=$1
```

Comment:
I think what you will actually want is to make your working directory be ${PROJECT_BINARY_DIR}/test/land/. Double check where these `mkdir` new directories are being made. They shouldn't be in the top level build/ but rather build/test/testrun